master/reference/cni-plugin/configuration.md

```diff
@@ -71,11 +71,26 @@ The following deprecated options are also supported
 
 When using Calico IPAM, the following flags determine what IP addresses should be assigned. NOTE: These flags are strings and not boolean values.
 
-* `assign_ipv4` (default `"true"`)
-* `assign_ipv6` (default `"false"`)
+* `assign_ipv4` (default: `"true"`)
+* `assign_ipv6` (default: `"false"`)
 
 A specific IP address can be chosen by using [`CNI_ARGS`](https://github.com/appc/cni/blob/master/SPEC.md#parameters) and setting `IP` to the desired value.
 
+Optionally, the list of possible IPv4 and IPv6 pools can also be specified via the following properties:
+
+* `ipv4_pools`: An array of CIDR strings (e.g. `"ipv4_pools": ["10.0.0.0/24", "20.0.0.0/16"]`)
+* `ipv6_pools`: An array of CIDR strings (e.g. `"ipv6_pools": ["2001:db8::1/120"]`)
+
+```markdown
+> *NOTE*
+>
+> `ipv6_pools` will be respected only when `assign_ipv6` is set to "true"
+```
+
+By default, all the IP Pools are used to assign IP addresses.
```

Comment:
"By default" can sometimes imply that the behavior is overrideable. Remove that. 

Also not a fan of saying "All the pools are used." How about:

"Calico IPAM will select an address from any of the specified IP pools."

Lastly, can we put this after the next sentence? I think explaining the error first is more important.